<launch>

<rosparam command="load" file="$(find roboteq_controller)/config/query.yaml" />

<node name = "roboteq_controller" pkg = "roboteq_controller" type = "roboteq_controller_node" output = "screen">


</node>

 
</launch>
